src/main/groovy/edu/oregonstate/mist/cardsapi/db/CardDAO.groovy

```diff
@@ -71,8 +35,29 @@ interface CardDAO extends Closeable {
     """)
     Card getCardById(@Bind("id") Integer id)
 
+    @SqlUpdate ("""
+        INSERT INTO CARDS (ID, TYPE_ID, NAME, COLOR_ID, RARITY_ID, ENERGY, DESCRIPTION)
+        VALUES (
+            (:id),
+            (SELECT TYPE_ID FROM CARD_TYPES WHERE TYPE = :type),
+            (:name),
+            (SELECT COLOR_ID FROM CARD_COLORS WHERE COLOR = :color),
+            (SELECT RARITY_ID FROM CARD_RARITIES WHERE RARITY = :rarity),
+            (:energy),
+            (:description)
+        )
+        """)
+    void postCard(@Bind("id") Integer id,
```

Comment:
Instead of binding all the individual objects, you could have one parameter as the Card object and use `@BindBean` to bind an object to the SQL query's bind variables